examples/gaussian_model_selection.py

```diff
@@ -0,0 +1,58 @@
+# A demo of using AIC / BIC for Model Selection
```

Comment:
Needs a docstring for the docs to build successfully (try running `make html` from the docs directory).